### PR TITLE
Feature: Support PHP 8.2

### PR DESCRIPTION
**Pull request by ghostwriter**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

- Update composer config `platform.php` to `8.0.99`
- Update composer `PHP` to `~8.0.0 || ~8.1.0 || ~8.2.0`
- Ignore PHP platform requirements via `.laminas-ci.json`

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,10 +19,13 @@
...
+        },
+        "platform": {
+            "php": "8.0.99"
         }
     },
     "require": {
-        "php": "^7.4 || ~8.0.0 || ~8.1.0"
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0"
     },
     "require-dev": {
```
Comment (Ocramius) [inline composer.json:30]: Needs a bump of PHPUnit to 9.5

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Comment by ghostwriter** [comment]
Note: ci-failure is unrelated.

> Schemas parser error : Failed to locate the main schema resource at 'vendor/phpunit/phpunit/phpunit.xsd'.

**Comment by Ocramius** [comment]
No, that's because PHPUnit < 9.5 didn't ship the schema at the current xsd location

**Review by Ocramius — APPROVED** [review]
Thanks @ghostwriter!

**Comment by ghostwriter** [comment]
> No, that's because PHPUnit < 9.5 didn't ship the schema at the current xsd location

Gotcha, thanks for the clarification.

**Pull request merged**